benchmarks/transport/mpi_p2p_bi_avail_mt.cpp

```diff
@@ -13,10 +13,38 @@
 #include <string.h>
 #include <atomic>
 
-#include <ghex/transport_layer/ucx/threads.hpp>
 #include <ghex/common/timer.hpp>
 #include "utils.hpp"
 
+#ifdef USE_OPENMP
+#include <omp.h>
+#define DO_PRAGMA(x) _Pragma(#x)
+
+#define GET_THREAD_NUM()  omp_get_thread_num()
```

Comment:
I was trying to rewrite the benchmark using thread primitives. Not sure if it makes sense... What do you think guys?

Reply:
@mbianco I wanted to keep the MPI benchmarks as stand-alone as possible, without any GHEX deps. there is the timer, but otherwise it should be clean now. These benchmarks are very simple, and I only need threads/no threads. Their usage is also quite limited (mostly to test the MPI with threading). So I would not spend too much time on them.